etna/whatson/models.py

```diff
@@ -373,11 +381,36 @@ class EventPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
         editable=False,
     )
 
-    registration_cost = models.IntegerField(
-        verbose_name=_("registration cost"),
+    min_price = models.IntegerField(
```

Comment:
Yeah, after that meeting we just had I was thinking it might be better to create a ticket type as well, but I think we can avoid that by using this. Ultimately we don't need to display the ticket types on our site, just the price range.

However, if we take the minimum price (if it's pay what you want/can, it seems to always be 0) and then take the maximum price (again, if it's pay what you can, it'll be X price) - we can then display it as `Free - £X` which is how Eventbrite shows it.